packages/web-frontend/src/containers/MeasureBar/index.js

```diff
@@ -149,10 +149,16 @@ export class MeasureBar extends Component {
       currentMeasure.datePickerLimits,
     );
 
+    const timePeriodEditable =
+      currentMeasure.timePeriodEditable !== undefined ? currentMeasure.timePeriodEditable : true; // defaults to true
```

Comment:
This is equivalent to your code but a bit simpler to read:
```suggestion
  const { timePeriodEditable = true } = currentMeasure;
```

Reply:
Ah, didn't know that one, nice. Updated.